core/app/models/spree/pricers/conservative_line_item_pricer.rb

```diff
@@ -0,0 +1,23 @@
+module Spree
+  module Pricers
+    class ConservativeLineItemPricer
+      class VariantMissing < StandardError; end
+      class CurrencyMissing < StandardError; end
+
+      attr_reader :line_item
+
+      def initialize(line_item)
+        @line_item = line_item
+      end
+
+      def price
```

Comment:
YARD: Sure. 
Money object: Makes sense, requires a bit of tweaking here and there, and also the other `PriceModifierLineItemPricer` to be reworked a little.